src/test/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializerTest.java

```diff
@@ -8,10 +8,9 @@
 
 package com.facebook.openwifirrm.optimizers.tpc;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import java.util.Map;
+import java.util.*;
```

Comment:
configure your IDE to never use wildcard imports, it's "generally" viewed as bad practice: https://stackoverflow.com/q/147454